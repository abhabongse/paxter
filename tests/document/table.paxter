@h1{Table}

@table[
    @table_header[
        {Name},
        {Age},
    ],
    @table_row[
        {@italic{PersonA} SurnameA},
        {25},
    ],
    @table_row[
        {@italic{PersonB} SurnameB},
        {48},
    ],
    @table_row[
        {
            First paragraph.

            Second paragraph.
        },
        {64},
    ],
]
