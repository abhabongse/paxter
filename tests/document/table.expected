<h1>Table</h1><table><tr><th>Name</th><th>Age</th></tr><tr><td><i>PersonA</i> SurnameA</td><td>25</td></tr><tr><td><i>PersonB</i> SurnameB</td><td>48</td></tr><tr><td><p>First paragraph.</p><p>Second paragraph.</p></td><td>64</td></tr></table>